website/docs/r/cosmosdb_postgresql_cluster.html.markdown

```diff
@@ -99,6 +99,8 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `earliest_restore_time` - The earliest restore point time (ISO8601 format) for the Azure Cosmos DB for PostgreSQL Cluster.
 
+* `server_names` - The name of the servers and its fully qualified domain name
```

Comment:
Since this is a block it should be 
```suggestion
* `servers` - A `servers` block as defined below.

---

A `servers` block exports the following:

* `fqdn` - The Fully Qualified Domain Name of the server.

* `name` - The name of the server.
```